lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -6,60 +6,16 @@ module Fastlane
 
   module Helper
     module FirebaseAppDistributionHelper
-      def testers_flag(params)
-        file_flag_if_supplied("--testers-file", "testers", params)
-      end
-
-      def groups_flag(params)
-        file_flag_if_supplied("--groups-file", "groups", params)
-      end
-
-      def release_notes_flag(params)
-        file_flag_if_supplied("--release-notes-file", "release_notes", params)
-      end
-
-      def file_flag_if_supplied(flag, param_name, params)
-        file = params["#{param_name}_file".to_sym]
-        file ||= file_for_contents(param_name.to_sym, params)
-
-        if file
-          return "#{flag} #{file}"
+      # Should we throw an exception if path is empty or not passed in? Check gradle plugin.
+      def get_value_from_value_or_file(value, path)
+        if value.nil? || value.empty? && !path.nil? || !path.empty?
+          UI.message(!path.empty?)
+          if File.exist?(path)
+            return File.open(path).read
+          end
+          UI.crash!("#{ErrorMessage::APK_NOT_FOUND}: #{path}")
```

Comment:
For readability I think it's preferable to wrap a failure case like this in the actual failure condition. This has a few benefits: it makes the error self-documenting, it takes the failure case out of the main flow of the code, and in this case it also removes the need for the early `return`:
```suggestion
          if !File.exist?(path)
            UI.crash!("#{ErrorMessage::APK_NOT_FOUND}: #{path}")
          end
          File.open(path).read
```

Another option is the one-liner: `UI.crash!("#{ErrorMessage::APK_NOT_FOUND}: #{path}") unless File.exist?(path)` which can be nice but in this case might be a little too long to be readable.